utils/build/docker/dotnet/poc.Dockerfile

```
@@ -35,4 +35,6 @@ ENV CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
 ENV DD_INTEGRATIONS=/opt/datadog/integrations.json
 ENV DD_DOTNET_TRACER_HOME=/opt/datadog
 
-CMD ["dotnet", "app.dll"]
+RUN echo "dotnet app.dll" >> ./app.sh
```

```suggestion
RUN echo "#!/bin/bash\ndotnet app.dll" > app.sh
```

And same for all other Dockerfiles